### PR TITLE
Customizing wizard labels

### PR DESCRIPTION
**Pull request by romalytvynenko**
Implementation of dashboard module for https://aventura.atlassian.net/browse/EDDBK-110

### CODE REVIEW

File: src/IteratorToArrayConvertCapable.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/**
+ * Trait IteratorToArrayConvertCapable, adds ability to convert an iterator to an array.
+ *
+ * @since [*next-version*]
+ */
+trait IteratorToArrayConvertCapable
+{
+    /* @since [*next-version*] */
+    use NormalizeIterableCapableTrait;
```
Comment (mecha) [inline src/IteratorToArrayConvertCapable.php]: This is not so ideal. The best approach for these kinds of traits is to add the method that you want to use in this trait as abstract. In this case, an abstract `_normalizeIterable()`. You can simply copy-paste the method from the `NormalizeIterableCapableTrait` and remove its body and mark it as abstract. Then, where it is used (such as in the `SettingsContainer`), also `use NormalizeIterableCapableTrait`, if not already used.

File: src/IteratorToArrayConvertCapable.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     * @since [*next-version*]
+     *
+     * @param array|stdClass|Traversable $iterator    The object to convert.
+     * @param callable                   $transformer Optional function that applies to value.
+     *
+     * @throws InvalidArgumentException If $iterator is not iterable.
+     *
+     * @return array Result of iterator to array transform.
+     */
+    protected function _iteratorToArrayRecursive($iterator, $transformer = null)
```
Comment (mecha) [inline src/IteratorToArrayConvertCapable.php]: Try to follow the naming convention that we already use, where the trait is named after the main method that it provides. Example: `IteratorToArrayRecursiveCapableTrait`.

**Review by mecha — CHANGES_REQUESTED** [review]

File: src/IteratorToArrayRecursiveCapable.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+use InvalidArgumentException;
+use stdClass;
+use Traversable;
+
+/**
+ * Trait IteratorToArrayConvertCapable, adds ability to convert an iterator to an array.
+ *
+ * @since [*next-version*]
+ */
+trait IteratorToArrayRecursiveCapable
```
Comment (mecha) [inline src/IteratorToArrayRecursiveCapable.php]: Much better! But you forgot the `Trait` suffix! :stuck_out_tongue:

**Review by mecha — CHANGES_REQUESTED** [review]

File: src/IteratorToArrayRecursiveCapable.php (hunk 3 above)
Comment (romalytvynenko) [reply src/IteratorToArrayRecursiveCapable.php]: Shame on me! Fixing.

**Review by mecha — APPROVED** [review]

**Pull request merged**